locales/en/add-email.ftl

```diff
@@ -19,4 +19,8 @@ add-email-address-input-label = Email address
 add-email-send-verification-button = Send verification link
 
 # $email is the newly added email address. $settings-href is the URL for the Settings page. HTML tags should not be translated, e.g. `<a>`
+# This string will be deprecated when the new Plus plan is live.
 add-email-verify-the-link = Verify the link sent to { $email } to add it to { -brand-fx-monitor }. Manage all email addresses in <a { $settings-href }>Settings</a>.
+# Variables:
+#   $email (string) - An email address submitted by the user for monitoring, e.g. `example@example.com`
+add-email-verify-the-link-2 = Verify the link sent to <b>{ $email }</b> to add it to { -brand-fx-monitor }.
```

Comment:
```suggestion
add-email-verify-the-link-2 = Verify the link sent to <b>{ $email }</b> to add it to { -brand-mozilla-monitor }.
```

Since this string is new, it might make sense to start using the new term? Obviously, #3934 would be a blocker for this, or you'd need to add the term here.

With that said, not a blocker, we can use the string with the previous term.

Reply:
That works for me, I added it in 734e350c5. Since this won't be visible until the redesign launches and hence #3934 is merged, I'm OK with it being broken for a bit.

Also, a heads-up that I apparently hadn't hit save, so I just added two more strings in 1c74aaeb6.

Reply:
Although I guess [this comment](https://github.com/mozilla/blurts-server/pull/3934#discussion_r1437433081) implies that maybe I should be using `fx-monitor` still?